### PR TITLE
CheckURL created.

### PR DESCRIPTION
**Pull request by mariacarmina**
Establish the connection with the checkURL endpoint from the provider repo. I have modified in DataServiceProvider class from data_provider package.

### CODE REVIEW

**Review by ssallam — CHANGES_REQUESTED** [review]
please remove all the `.idea/` files from the repo

File: ocean_lib/data_provider/data_service_provider.py [hunk 1]
```diff
@@ -10,6 +10,8 @@
 from collections import namedtuple
 from json import JSONDecodeError
 
+from ocean_lib.data_provider import data_service_provider
```
Comment (ssallam) [inline ocean_lib/data_provider/data_service_provider.py:13]: what is this doing? we are in the data_provider already!

**Comment by ssallam** [comment]
It seems we have two PRs doing similar things: #78

**Comment by calina-c** [comment]
Duplicated by #78.

**Pull request closed without merging**